pages/_app.tsx

```diff
@@ -5,6 +5,14 @@ import { createTheme, ThemeProvider, CssBaseline } from '@mui/material';
 import { Roboto } from 'next/font/google';
 import type { AppProps } from 'next/app';
 import { grey } from '@mui/material/colors';
+var getConfig = require("next/config");
+const { serverRuntimeConfig } = getConfig();
+
+let style = {};
+(async () => {
```

Comment:
If this is being added because `serverRuntimeConfig` does not allow async functions then the original use case of the next/config losses its value.

Reply:
We changed the serverRuntimeConfig to PublicRuntimeConfig since we cannot apply the styles from the browser since its only available in the server side